plugins/3MFWriter/SettingsExportGroup.py

```diff
@@ -32,6 +33,14 @@ def name(self):
     def settings(self):
         return self._settings
 
+    @pyqtProperty(list, constant=True)
+    def visibleSettings(self):
+        if self._visible_settings == []:
+            for item in self._settings:
+                if item.isVisible:
+                    self._visible_settings.append(item)
```

Comment:
I have a slight preference for writing this as the following as it is more concise
```suggestion
            self._visible_settings = list(filter(lambda item : item.isVisible, self._settings))
```
however, I get that that is personal preference.